programs/utilities/sscoreboard.sc

```diff
@@ -0,0 +1,90 @@
+//statistic scoreboard interface by Lokdora
+//dynamically create scoreboard based on statistics and assigned it with correct value (when player is online)
+//options to distinguish bots from other players
+//due to the scoreboard value cannot be assigned when player is offline, it is better to use this script to create scoreboard objectives for mid/long-term use (or singleplayer world).
+
+
+__config() -> {
+    'stay_loaded' -> 'true',
+	'scope' -> 'global',
+	'commands' -> {
+		'' -> 'help',
+		'clear' -> 'clear',
+		'<stat> <displayName>' -> ['create','sidebar',false],
+		'<stat> <displayName> <displaySlot>' -> ['create',false],
+		'<stat> <displayName> <displaySlot> <includeBots>' -> 'create'
+	},
+	'arguments' -> {
+		'stat' -> {'type' -> 'term'},

```

Comment:
do you think there's any way to possibly figure these out so you can do command suggestions? If you look [here](https://github.com/gnembon/fabric-carpet/blob/master/docs/scarpet/Full.md#command-argument-types) and scroll down, you may be able to use the 'objective' or 'criterion' argument types.

Reply:
oh, i see now

Reply:
I don't know what is wrong, but after i changing the term to criterion the function just stopped working. I have done some tests and i found that the create function itself still works when i use the "script in" command to run it, but when i use the generated command it said *An unexpected error occurred trying to execute that command*, and no suggestions pop out  when i am tying the commands in.
If I didn't do something wrong by accident, I believe the only way to do the proper command suggestion is to manually type everything in the script which is too tedious. Although I think the behavior of ‘criterion' type might be a bug, which I am not sure, I decided not to make a change to the script now.